book/http.md

```diff
@@ -659,8 +658,8 @@ servers seem to use together with compression.
 Displaying the HTML
 ===================
 
-The HTML code in the body defines the content you see in your browser
-window when you go to <http://example.org/index.html>. I'll be
+The HTML code in the response body defines the content you see in your browser
+window when you go to `<http://example.org/index.html>`. I'll be
```

Comment:
Here the link should, I think, be distinguished somehow. Typewriter font is fine, as is an underline. I'm not sure if this actually renders right.